my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
+pyplot.figure()
+pyplot.plot(np.arange(1024), col1)
+mean = np.mean(col1)
+pyplot.axhline(mean)
+pyplot.legend(['Column 20', 'Mean'])
```

Comment:
It's better to label the individual objects; it's not clear what this command does right now.